### DISCUSSION TITLE
Is it possible to trigger the deployment of an auto deploying manifest without making changes to a manifest?

### DISCUSSION THREAD

**Discussion by cskwrd** [k3s-io/k3s#6793] [Q&A] [answered]
I need to redeploy an auto deploying manifest. Is there a command I can run to trigger redeployment or is the only way to upload a changed file?

I tried `touch -r <path to manifest> <path to manifest>`, but that did not seem to have any effect.

**Comment by brandond** [comment, marked as answer]
No, the actual content needs to change. The content can be just comment or whitespace changes though.
